libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -734,6 +734,7 @@ dictionary RecommendedFees {
 interface InputType {
     BitcoinAddress(BitcoinAddressData address);
     Bolt11(LNInvoice invoice);
+    Bolt12(string offer);
```

Comment:
I think we should have here the parsed LNOffer (same as we have for other types

Reply:
Addressed: https://github.com/breez/breez-sdk-greenlight/pull/1121#issuecomment-2472855649